latexify/core.py

```diff
@@ -54,8 +55,10 @@ def visit_Module(self, node):
   def visit_FunctionDef(self, node):
     name_str = r'\mathrm{' + str(node.name) + '}'
     arg_strs = [self._parse_math_symbols(str(arg.arg)) for arg in node.args.args]
-    body_str = self.visit(node.body[0])
-    return name_str + '(' + ', '.join(arg_strs) + r')\triangleq ' + body_str
+    for body_item in node.body[:-1]:
```

Comment:
We need more strict check: this visitor should only allow a series of assignments and denies other statements (e.g., if-blocks, which confuses which variable is assigned, and its content).